riscv/src/continuations/bootloader.rs

```diff
@@ -73,22 +73,22 @@ pub fn bootloader_preamble() -> String {
     for (i, reg) in REGISTER_NAMES.iter().enumerate() {
         let reg = reg.strip_prefix("main.").unwrap();
         preamble.push_str(&format!(
-            "    public initial_{reg} = main_bootloader_inputs.value({i});\n"
+            "    //public initial_{reg} = main_bootloader_inputs.value({i});\n"
```

Comment:
Maybe create an issue for publics not working with composite backend, and mention it here?